code/modules/mob/language.dm

```diff
@@ -508,6 +508,17 @@
 	drone_only = 1
 	follow = 1
 
+/datum/language/swarmer
+	name = "Swarmer"
+	desc = "A heavily encoded alien binary pattern."
+	speech_verb = "tones"
+	ask_verb = "tones"
+	exclaim_verb = "tones"
+	colour = "say_quote"
+	key = "zw"//Zwarmer...Or Zerg!
```

Comment:
did you test this? I'm not sure if dual key language keys work or not---I know they didn't, at one point, but that may be fixed now.